test/integration/forms/form.js

```diff
@@ -1,21 +1,30 @@
 context('Patient Form', function() {
   specify('routing to form', function() {
+    cy
```

Comment:
is this visit needed?  If so that may be a problem

Reply:
nope, sure isn't!